mlir/test/Integration/Dialect/Linalg/CPU/pack-unpack-mmt4d.mlir

```diff
@@ -0,0 +1,172 @@
+// DEFINE: %{compile} =  mlir-opt %s \
+// DEFINE:    -transform-interpreter -test-transform-dialect-erase-schedule \
+// DEFINE:    -one-shot-bufferize="bufferize-function-boundaries" \
+// DEFINE:    -buffer-deallocation-pipeline  -cse -canonicalize -convert-vector-to-scf -test-lower-to-llvm
```

Comment:
`-convert-vector-to-scf` can be removed, it's the first pass in `-test-lower-to-llvm`. Also, replace double space before `-cse` with single space.

```suggestion
// DEFINE:    -buffer-deallocation-pipeline -cse -canonicalize -test-lower-to-llvm
```